### ISSUE TITLE
Badge on wiki tab that displays wiki pages count.

### ISSUE THREAD

**Issue by adisos** [isaacs/github#758]
I would like a badge on wiki tab that displays wiki pages count.
![image](https://cloud.githubusercontent.com/assets/592907/18206566/221ef68a-7149-11e6-97dd-cb9ab7d03650.png)

**Comment by winek** [comment]
This. Sometimes if not satisfied with the README, I click the wiki tab. Blank page. Wasted seconds. Frustration doubled. The wiki link is useless and misleading if it has no single commit.

**Comment by adisos** [comment]
@winek 
Yeah, I like idea that if there is no wiki content - better hide link
